src/Assignment-2/assignment2.html

```diff
@@ -8,21 +8,154 @@
 	<script src="bundle.js" type="text/javascript"></script>
 </head>
 
-<body>
+<body style="background-color:lavender;">
 	<div id="container">
+		
+		<header style="border-style: solid; max-height: 100px; height: 100px; border-bottom-right-radius:15px; border-bottom-left-radius: 15px ; background-color:rgb(132, 150, 161);border-color: rgb(62, 118, 119);">
+			<div class="header-img">
+			  <img src="https://www.whatsappimages.in/wp-content/uploads/2020/06/Whatsapp-DP-Images-8-300x300.jpg">
+			</div>
+		
+			<div class="header-input">
+				<input type="text" placeholder="Search" name="Search"/>
+			</div>
+		
+			<nav>
+				<h2> 
+					<a href="/Section 1/" style="color: rgb(15, 92, 117);">Section 1</a>
+					<a href="/Section 2/" style="color: rgb(15, 92, 117);">Section 2</a>
+					<a href="/Section 3/" style="color: rgb(15, 92, 117);">Section 3</a>
+					<a href="/Section 4/" style="color: rgb(15, 92, 117);">Section 4</a>
+				</h2>
+			</nav>
+		
+		  </header>	
+		  <div class="page-body">
+			<aside>
+			  <div>
+				<ul>
+				  <h3>
```

Comment:
inside ul tag use li tag directly no need of h3 tag again. use css propertiles in div tag of aside for formatting